### ISSUE TITLE
Blacklisting Maximized windows

### ISSUE THREAD

**Issue by actforjason** [RamonUnch/AltSnap#141]
`Enable on full screen windows` is unchecked，but AltSnap still works on fullscreen windows，such as Pycharm.
![image](https://user-images.githubusercontent.com/57058325/148630501-048af15d-2afb-46fe-9cd4-7df2e4b94652.png)

**Comment by RamonUnch** [comment]
Could you use the identify window on the full-screen Pycharm window and show me the results?
For a window to be considered full-screen it needs to exactly match the monitor size (ie: no task-bar visible) and it also mush have no caption, otherwise all maximized windows would be blacklisted.

**Comment by actforjason** [comment]
You can also reproduce it easily on Chrome.
![动画](https://user-images.githubusercontent.com/57058325/148641292-7c5d3212-0d75-4ba4-91ef-ee7635ef663e.gif)

**Comment by RamonUnch** [comment]
I cant reproduce, (I do not have access to win10/11 for now though).
How do you get the title-bar still visible in full-screen mode?
It seems the dimensions of your screenshot is 1919x1079, so 1 pixel shy of full-screen, maybe that is the problem, the question is why is that? I will try to add a few pixels tolerance for full-screen detection.

Also if the windows has a caption then it is NOT considered "full-screen", so from your screenshot it is not a bug. Actually it just looks maximized.

I will investigate more, but could you provide simply the content of the Identify Window? 
It can be obtained from the Blacklist tab:
1) Open AltSnap config to the Blacklist tab and set it "Always On Top",
2) Open your problematic program set it fullscreen,
3) Click on the cursor in AltSnap's config dialog, then on the problematic window.
4) send me a screenshot of the result.

**Comment by Ichisich** [comment]
@actforjason 
In your example, Chrome seems to be just maximized, not in its fullscreen-state.
Press F11 and test it again.

**Comment by actforjason** [comment]
Oh sorry,  I got mixed up. But, is there a way to disable altsnap when windows are maximized?

**Comment by RamonUnch** [comment]
@actforjason 
I will add it, it is a good idea.

**Comment by RamonUnch** [comment]
Here is a version of AltSnap that has the option via the .ini file only (middle click the tray icon).
in the [Advanced] section of AltSnap.ini:
```
BLMaximized=0
; Set to 1 to disable AltSnap on all Maximized windows.
; If set to 0 (default) maximized windows will be restored by AltSnap.
; Not the same thing than "fullscreen" window but looks quite similar.
```
[AltSnap-installers.zip](https://github.com/RamonUnch/AltSnap/files/7834060/AltSnap-installers.zip)
[AltSnap1.49bin_x64.zip](https://github.com/RamonUnch/AltSnap/files/7834061/AltSnap1.49bin_x64.zip)
[AltSnap1.49bin.zip](https://github.com/RamonUnch/AltSnap/files/7834062/AltSnap1.49bin.zip)

**Comment by RamonUnch** [comment]
Included in the 1.49 release
